modules/oc-mirror-command-reference-v2-delete.adoc

```diff
@@ -0,0 +1,72 @@
+// Module included in the following assemblies:
+//
+// * installing/disconnected_install/installing-mirroring-disconnected-v2.adoc
+
+
+:_mod-docs-content-type: REFERENCE
+[id="oc-mirror-command-reference-delete-v2_{context}"]
+= Command reference for deleting images
+
+The following tables describe the `oc mirror` subcommands and flags for deleting images:
+
+.Subcommands and flags for the deleting images
+[cols="1,2",options="header"]
+|===
+|Subcommand
+|Description
+
+|`--authfile string `
+|Path of the authentication file. The default is `${XDG_RUNTIME_DIR}/containers/auth.json`.
+
+|`--cache-dir string` 
+|oc-mirror cache directory location. The default is `$HOME`.
+
+|`-c`, `--config string` 
+|Path to the delete imageset configuration file.
+
+|`--delete-id string` 
```

Comment:
i am not sure if we have used `--delete-id` flag and tested the functionality extensively, @zhouying7780 ??

Reply:
I never tested this flag.